sdks/python/apache_beam/testing/util.py

```diff
@@ -113,14 +113,19 @@ def matcher(elements):
 # However the sorting does not go beyond top level therefore [1,2] and [2,1]
 # are considered equal and [[1,2]] and [[2,1]] are not.
 def equal_to(expected):
-  expected = list(expected)
 
   def _equal(actual):
-    sorted_expected = sorted(expected)
-    sorted_actual = sorted(actual)
-    if sorted_expected != sorted_actual:
+    expected_list = list(expected)
+    for element in actual:
+      try:
+        expected_list.remove(element)
```

Comment:
This no longer does what the above comment about sorting. Please update the comment. (or do it in a follow up)

Reply:
Updated in #7022. I'll rebase this one.